### PR TITLE
Added tests to addHiddenValues

### PR DESCRIPTION
**Pull request by orendig**

### CODE REVIEW

File: packages/core/test/workspace/hidden_values.test.ts [hunk 1]
```diff
@@ -142,4 +152,57 @@ describe('hidden_values.ts', () => {
...
+      workspaceInstance.value.numHidden = 11111
+
+
+      const stateInstance = hiddenInstance.clone()
+
+      let resp: Element[]
+      let instanceAfterHiddenAddition: InstanceElement
+      beforeAll(async () => {
+        resp = addHiddenValues([workspaceInstance], [stateInstance])
+        instanceAfterHiddenAddition = resp[0] as InstanceElement
```
Comment (adamro) [inline packages/core/test/workspace/hidden_values.test.ts:178]: maybe also check resp's length is only 1? (and generally that it "merges" the right elements so the number of elements returned is equal to the number of unique elemID fullNames [guessing the logic here])

File: packages/core/test/workspace/hidden_values.test.ts [hunk 2]
```diff
@@ -142,4 +152,57 @@ describe('hidden_values.ts', () => {
...
+        expect(workspaceInstance.value.notHidden)
+          .toEqual(instanceAfterHiddenAddition.value.notHidden)
+        expect(workspaceInstance.value.reg)
+          .toEqual(instanceAfterHiddenAddition.value.reg)
+        expect(workspaceInstance.value.listOfObjects[0].num)
+          .toEqual(instanceAfterHiddenAddition.value.listOfObjects[0].num)
+      })
+
+      it('should not done in-place', () => {
+        expect(stateInstance.isEqual(hiddenInstance)).toBeTruthy()
```
Comment (adamro) [inline packages/core/test/workspace/hidden_values.test.ts:204]: if this is the case why not use `ReadonlyArray` for `stateElements` in `addHiddenValues`?

Comment (orendig) [reply packages/core/test/workspace/hidden_values.test.ts:204]: That’s really good question

File: packages/core/test/workspace/hidden_values.test.ts (hunk 1 above)
Comment (orendig) [reply packages/core/test/workspace/hidden_values.test.ts:178]: added

**Pull request merged**